text/3804-cfg-alias.md

```diff
@@ -0,0 +1,165 @@
+- Feature Name: `cfg_alias`
+- Start Date: 2025-04-23
+- RFC PR: [rust-lang/rfcs#3804](https://github.com/rust-lang/rfcs/pull/3804)
+- Rust Issue:
+  [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+
+[summary]: #summary
+
+This RFC introduces a way to name configuration predicates for easy reuse
+throughout a crate.
+
+```rust
+#![cfg_alias(
+    x86_linux,
+    all(any(target_arch = "x86", target_arch = "x86_64"), target_os = "linux")
+)]
+
+#[cfg(x86_linux)]
+fn foo() { /* ... */ }
+
+#[cfg(not(x86_linux))]
+fn foo() { /* ... */ }
+```
+
+# Motivation
```

Comment:
another reason to have something like this is when you have some WIP code that you want to temporarily disable, you can have `#![cfg_alias(todo, false)]` which allows you to `#[cfg(todo)]` any code you want to temporarily disable, while still being readily distinguishable from normal `#[cfg(...)]` that shouldn't get removed later.

Reply:
Added this example, thanks